Tests/Acceptance/DoctrineRefreshTokenManagerTest.php

```diff
@@ -55,16 +55,16 @@ public function testClearExpired(): void
     private function buildClearExpiredTestData(Client $client): array
     {
         $validRefreshTokens = [
-            $this->buildRefreshToken('1111', '+1 day', $client),
-            $this->buildRefreshToken('2222', '+1 hour', $client),
-            $this->buildRefreshToken('3333', '+1 second', $client),
-            $this->buildRefreshToken('4444', 'now', $client),
+            $this->buildRefreshToken($client, '1111', '+1 day'),
```

Comment:
@franjo-zadelj-trikoder What is the purpose of the signature change?

Reply:
same as above